src/app/page.tsx

```diff
@@ -75,14 +75,17 @@ export default function Home() {
             <h2>Results of : '{currentSearchQuery}'</h2>
           </>
         )}
-        <Loader isLoading={isLoading} />
-        {articles?.length === 0 && (
-          <div className={Styles.nothing}>
-            <Image width={200} height={200} alt='No svg' src={'/sadSmile.svg'}></Image>
-            <h2>
-              Not Found... <hr /> Try again later
-            </h2>
-          </div>
+        {isLoading ? (
+          <Loader />
+        ) : (
+          articles?.length === 0 && (
```

Comment:
You don’t need optional chaining here because now you always have an array so use `!articles.length`